src/Skeletor/Manager/PackageManager.php

```diff
@@ -163,6 +163,46 @@ public function configure(Package $package, Framework $activeFramework)
     {
         if (!$this->options['dryRun']) {
             $package->configure($activeFramework);
+
+            // Update .env files
+            $this->updateEnvironmentVariables($package);
+        }
+    }
+
+    /**
+     * @param  Package $package
+     */
+    public function publishConfig(Package $package)
+    {
+        if (!$this->options['dryRun']) {
+            $output = $package->publishConfig();
+
+            $this->cli->br()->output('Publishing configuration');
+            $this->cli->output($output);
+        }
+    }
+
+    /**
+     * Update the .env and .env.example files with the environment variables
+     * that are specified for the package.
+     *
+     * @param  Package $package
+     */
+    private function updateEnvironmentVariables(Package $package)
+    {
+        // Write the package environment variables to the .env files
+        if ($package->hasEnvironmentVariables() === true) {
+            $this->cli->br()->output('Writing environment variables');
+
+            $envVariables = $package->getEnvironmentVariables();
+            $lines = "\n";
```

Comment:
I'd use the PHP_EOL constant, to be sure it is compatible with all operating systems.